src/utils/rewards.js

```diff
@@ -15,6 +15,13 @@ export const convertWPRtoAPY = (wpr) =>{
   return apy_passiv
 }
 
+export const convertWPRtoDoubleAPY = (wpr) =>{
+  const weeks = 52
+  const double_apr = wpr * 2;
+  const double_apy = ((1 + double_apr / weeks) ** 52 - 1) * 100
+  return double_apy
+}
+
```

Comment:
we already have the `convertWPRtoAPY(wpr)` doing the same thing, can we use that one? Just pass it wpr * 2 as param.
The current formula is also wrong, the double_apr should be `double_apr=wpr * 2 * weeks`